scripts/health_check.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+#example usage: health_check.sh --network=main --min_height=52000 --version=2.1.0 18.136.37.63
+set -eo pipefail
+
+declare -A network_genesis_hashmap
```

Comment:
This is bash 4, OSX still ships bash 3

I wonder should we support bash 3 and OSX. @michalbagrowski what's your opinion ?

Reply:
```↪ ./scripts/health_check.sh --network=main --min_height=54000 --version=2.2.0 18.136.37.63
./scripts/health_check.sh: line 5: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]```